public/js/libs/area_chart.js

```diff
@@ -98,6 +99,23 @@ export class AreaChart extends GraphLayout {
             .call(d3.axisBottom(this.xScale).tickSize(-this.height).tickFormat(""));
     }
 
+    displayYLimit() {
```

Comment:
It would probably be better if we added the limit to the last tick in Y-axis. With different formatting to distinguish from the rest. Otherwise, the current design makes the graph look too busy.

Reply:
Updated, thanks.